app/setup/on_start.py

```diff
@@ -41,3 +42,14 @@ async def update_api_instance():
         raise e
     finally:
         api_instance.reset()
+
+
+async def clear_dead_api_instances():
+    try:
+        clearing_result = await storage.driver.api_instance.remove_dead_instances()
+        logger.info(f"Performed dead API instances removal. Total of {clearing_result['deleted']} dead instances "
+                    f"removed.")
+
+    except StorageException as e:
+        logger.error(f"Dead API instances could not be removed due to an ERROR `{str(e)}`")
```

Comment:
Did you check if this did not stop the server?

Reply:
yes, it does not do so